xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -19,8 +19,18 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides a high-performance API for forward-only, read-only access to the UTF-8 encoded JSON text. It processes the text sequentially with no caching and by default adheres strictly to the <see href="https://tools.ietf.org/html/rfc8259">JSON RFC</see>.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[  
+
+## Remarks
```

Comment:
These remarks have a `## Remarks`.  JsonWriterState's remarks don't...

Reply:
Yeah, I switched this one to use markdown so we have the extra heading.